jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -285,10 +285,34 @@ public static void logMessage(String string1, int int1, String string2, String s
 		}
 	}
 
+	/**
+	 * Print the information about a throwable, including the exact class,
+	 * message, and stack strace.
+	 * @param msg User supplied message
+	 * @param thrown throwable
+	 */
+	public synchronized static void logMessage(String msg, Throwable thrown) {
+		tracepoint(TRACEPOINT_STATUS_LOGGING, msg);
+		@SuppressWarnings("resource")
+		PrintStream log = getLogStream();
+		printLogMessageHeader(log);
+		log.println("Throwable: "+thrown.getClass().getName()); //$NON-NLS-1$
+		log.println("message: "+thrown.getMessage()); //$NON-NLS-1$
+		thrown.printStackTrace(log);
```

Comment:
The msg parameter isn't printed to the log. Seems to me the printStackTrace() will include the throwable and message so the previous two lines aren't needed.